var/spack/repos/builtin/packages/r-libpressio/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class RLibpressio(RPackage):
+    """R package for libpressio"""
+
+    homepage = "https://github.com/robertu94/libpressio-r"
+    url = "https://github.com/robertu94/libpressio-r/archive/0.0.1.tar.gz"
+
+    # notify when the package is updated.
```

Comment:
(minor) I think this comment should be removed.

Reply:
fixed